.github/ISSUE_TEMPLATE/recipe-request.md

```diff
@@ -4,9 +4,14 @@ about: Request a recipe be added this repo
 title: ''
 labels: recipe-request
 assignees: ''
-
 ---
 
+<!---
+Before you submit this recipe request, check if there are any duplicate ["Recipe Request" issues](https://github.com/JordanMartinez/purescript-cookbook/issues?q=is%3Aissue+is%3Aopen+label%3Arecipe-request) or [PRs](https://github.com/JordanMartinez/purescript-cookbook/pulls).
+
+If you'd like to work on implementing this recipe, assign yourself and let us know when to check back in to make sure it's not abandoned.
```

Comment:
Figured this note is better to include directly in the issue form, rather than the contributing doc.

Reply:
Makes sense. Why is this wrapped in a comment?

Reply:
The comment wrapping prevents it from appearing in the issue if it's not deleted.

Reply:
Smart!